### ISSUE TITLE
Accessibility - Exploitation Types Screen - TalkBack Reads Tab Text  in InCorrect Order

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#49]
When using TalkBack to read the screen, it read the tab order incorrectly. It first read:
- All
- Forced Child Labor
- Forced Labor
- Child Labor

TalkBack should have read the tabs in the following order:
- All
- Child Labor
- Forced Labor
- Forced Child Labor

**Comment by aquilent-dgemoets** [comment]
Default Android Tab views do not support icons, so to meet this requirement, a third-party custom tab view was used.  Unfortunately, this tab view was not optimized for accessibility, and based on initial analysis, it would be difficult to correct its reading by TalkBack.  Fortunately, "All" is still the first one read.

**Comment by jfc3-dol** [comment]
It's confusing for those using a screen reader like TalkBack to read along because they have read or comprehension issues and see the focus skip around the screen in the tabs section.

**Comment by aquilent-dgemoets** [comment]
To address this issue, we would either need to add proper accessibility to the third-party widget, or change the UI in some other way-- perhaps removing tabs altogether and changing them to four Icons which would update the list below it.  Either way, this work would need to be spec'ed out by the developer and triaged with other tasks.

**Comment by jfc3-dol** [comment]
I would think there is a third-party or another widget that reads tabs from left to right already out there.

**Comment by aquilent-dgemoets** [comment]
The standard Android tabs are read from left to right, but they do not support icons.  The third-party tab widget that supports icons has problems with reading order.

A better alternative to my suggestion above ("removing tabs altogether and changing them to four Icons which would update the list below it") would be to replace the tabs with a dropdown much like the Exploitation Types dropdown on the Individual Goods screen.  It would also actually be more consistent as all exploitation type filters would work the same way (via dropdown).

Or we can try to add proper accessibility (reading order) to the current third-party tabs-with-icons widget.  The developer is investigating/estimating both options.

**Comment by jfc3-dol** [comment]
Making this a drop down list removes the issue with the different tabs, so closing the issue.
